test/widgets/drive_detail_page_test.dart

```diff
@@ -0,0 +1,164 @@
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+import 'package:mockito/mockito.dart';
+import 'package:motis_mitfahr_app/drives/models/drive.dart';
+import 'package:motis_mitfahr_app/drives/pages/drive_chat_page.dart';
+import 'package:motis_mitfahr_app/drives/pages/drive_detail_page.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:motis_mitfahr_app/rides/models/ride.dart';
+
+import '../util/factories/drive_factory.dart';
+import '../util/factories/ride_factory.dart';
+import '../util/mock_server.dart';
+import '../util/mock_server.mocks.dart';
+import '../util/pump_material.dart';
+
+void main() {
```

Comment:
what is with the riders and pending rides, can we add a test that they are there also? 
and also that a ride that is cancelledbyRider does not appear?

Reply:
Yes that would probably be a good idea 👍